test/__init__.py

```diff
@@ -53,11 +54,24 @@ def run_role_meta_main(self, meta_main_text):
         return results
 
 
-def run_ansible_lint(cwd, role_path=None, bin=None, env=None):
-    """Run ansible-lint on a given path and returns its output."""
-    command = '{} -v {}'.format(
-        bin or (sys.executable + " -m ansiblelint"),
-        role_path or "")
+def run_ansible_lint(
+        cwd: os.PathLike = None,
+        role_path: os.PathLike = None,
+        bin: str = None,
+        env: Dict[str, str] = None):
+    """Run ansible-lint on a given path and returns its output.
+
+    If "." is passed to cwd, the root of the ansible-lint source repository will be used.
+    """
+    if bin:
+        command = [bin]
+    else:
+        command = [sys.executable, "-m", "ansiblelint"]
+    if role_path:
+        command.append(role_path)
+
+    if cwd == ".":
+        cwd = os.Path(__file__).parents[1]
 
     result, err = subprocess.Popen(
         [command],
```

Comment:
Don't forget to fix this, otherwise, it's a nested list
```suggestion
        command,
```